src/fides/api/service/saas_request/override_implementations/interruptable_request.py

```diff
@@ -0,0 +1,30 @@
+import os
```

Comment:
If this file is truly needed, you should rename it to `interruptible_request.py` (i, not a)